cluster/gce/upgrade.sh

```diff
@@ -33,7 +33,7 @@ source "${KUBE_ROOT}/cluster/kube-util.sh"
 function usage() {
   echo "!!! EXPERIMENTAL !!!"
   echo ""
-  echo "${0} [-M|-N|-P] -l -o | <version number or publication>"
+  echo "${0} [-M|-N|-P] [-o] -l | <version number or publication>"
```

Comment:
Brackes mean optional, pipe means or. We can pass one of `-M, -N, -P`, but no matter what we choose, we can pass -o as well.